src/wp-includes/class-wp-image-editor.php

```diff
@@ -378,8 +402,9 @@ protected function get_output_format( $filename = null, $mime_type = null ) {
 		 * }
 		 * @param string $filename  Path to the image.
 		 * @param string $mime_type The source image mime type.
+		 * @param string $size_name The image size name to create, or empty string if not set.
 		 */
-		$output_format = apply_filters( 'image_editor_output_format', array(), $filename, $mime_type );
+		$output_format = apply_filters( 'image_editor_output_format', array(), $filename, $mime_type, $this->size_name );
```

Comment:
Great! This will also set us up to include size_name in the `wp_editor_set_quality` filter